docs/01-app/02-guides/incremental-static-regeneration.mdx

```diff
@@ -67,21 +68,22 @@ export default async function Page({
 }
 ```
 
-```jsx filename="app/blog/[id]/page.js" switcher
+```jsx filename="app/blog/[id]/page.jsx" switcher
 // Next.js will invalidate the cache when a
 // request comes in, at most once every 60 seconds.
 export const revalidate = 60
 
 // We'll prerender only the params from `generateStaticParams` at build time.
 // If a request comes in for a path that hasn't been generated,
 // Next.js will server-render the page on-demand.
-export const dynamicParams = true // or false, to 404 on unknown paths
+export const dynamicParams = true // setting to false will 404 on unknown paths instead
```

Comment:
Since we are trying to improve the docs here, I am a bit torn.

`true` is the default value - maybe it is okay to leave it as is

```tsx
// We'll prerender only the params from `generateStaticParams` at build time.
// Using `dynamicParams` set to true (default value) If a request comes in 
// for a path that was not generated at build time 
// Next.js will attempt to server-render the page on-demand.
// Use `dynamicParams` set to false to response with 404 instead.
export const dynamicParams = true
```

That being said, this feels like information that perhaps shouldn't be in a comment, but appended to the 6th point in the "Here's how this example works:" list.